app/src/app/(public)/properties/schedule/[id]/page.tsx

```
@@ -0,0 +1,58 @@
+  const urlParameters = await searchParams;
+
+  const formValues = {
+    check_in: urlParameters.check_in ? new Date(urlParameters.check_in) : TODAY,
+    check_out: urlParameters.check_out
+      ? new Date(urlParameters.check_out)
+      : TOMORROW,
+  };
+
+  const propertyId = Number(id);
+
+  if (isNaN(propertyId)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)